frontend/src/features/admin-form/responses/ResponsesPage/storage/UnlockedResponses/ResponsesTable/ResponsesTable.tsx

```diff
@@ -99,8 +99,9 @@ const PAYMENT_COLUMNS: Column<ResponseColumnData>[] = [
       }
       return payments.payoutDate
     },
-    minWidth: 50,
-    width: 150,
+    minWidth: 200,
+    width: 200,
```

Comment:
Thanks to increasing this too Ken, so we want to keep each row as one line without wraps right?

May I suggest adding another manual test for this too
- [ ] ensure payout date does not wrap (edit html: Tue, 19 Sep 2023)